frame/contracts/src/tests.rs

```diff
@@ -1718,3 +1718,8 @@ fn crypto_hashes() {
 		}
 	})
 }
+
+fn load_wasm(file_name: &str) -> String {
+	let path = ["src/tests/", file_name].concat();
```

Comment:
I personally wouldn't put it under `src/tests` but rather just under `tests` since those files are not code source files.

Reply:
Done.